libs/ui/lib/checkbox/Checkbox.tsx

```diff
@@ -20,7 +20,7 @@ const inputStyle = `
   appearance-none border border-default bg-default h-4 w-4 rounded-sm absolute left-0 outline-none
   disabled:cursor-not-allowed
   hover:border-hover hover:cursor-pointer
-  checked:bg-accent-secondary checked:border-accent-secondary checked:hover:border-accent
+  checked:bg-accent-secondary checked:border-accent-secondary checked:hover:border-accent [&:checked+svg]:block
```

Comment:
Not really affected by what we're doing here but felt like a reasonable QoL change – this means that the checkbox still shows up if the `checked` prop is not being controlled.